### PR TITLE
Allows ghosts to dab

### PR DESCRIPTION
**Pull request by Superlagg**
[FEATURE]

## About the PR 

Ghosts can now dab. Even plays the dab animation too.

Dabbing also has a 0% chance of being sent straight to hell.

## Why's this needed? 

Death wasn't horrible enough.

## Changelog

```
(u)Superlagg:
(*)Ghosts can now dab.
```

### CODE REVIEW

**Comment by Kyle2143** [comment]
I believe that we recently have removed all official, non-admin ways to enter hell for the time being. I'm not sure that we should add it back here.

**Comment by Gerhazo** [comment]
We don't have many ways to punish ghosts for crimes.

**Comment by Superlagg** [comment]
> I believe that we recently have removed all official, non-admin ways to enter hell for the time being. I'm not sure that we should add it back here.

I agree, ghosts should be able to commit dab without consequence. Dabmnation removed!

File: code/mob/dead.dm [hunk 1]
```diff
@@ -166,3 +182,48 @@
...
+	sleep(0.1) //so the direction setting actually takes place
+	world << torso
+	world << right_arm
+	world << left_arm
+	torso.plane = PLANE_DEFAULT
+	right_arm.plane = PLANE_DEFAULT
+	left_arm.plane = PLANE_DEFAULT
+	/*torso.loc = get_turf(O)
+	right_arm.loc = get_turf(O)
+	left_arm.loc = get_turf(O)*/
```
Comment (moonlol) [inline code/mob/dead.dm:213]:
```suggestion
```
these seem unused id suggest removing them for code cleanliness

**Pull request closed without merging**